docker/docker.md

```diff
@@ -27,16 +27,37 @@ Before running the java-tron service, make sure some ports on your local machine
 - `50051`: used by the GRPC based API
 - `18888`: TCP and UDP, used by the P2P protocol running the network
 
-then start the java-tron service with the `--run` parameter
+#### Full node on the main  network
+
 ```shell
-$ sh docker.sh --run
+$ sh docker.sh --run --net main
 ```
 or you can use `-p` to customize  the port mapping, more custom parameters, please refer to [Options](#Options)
 
 ```shell
-$ sh docker.sh --run  -p 8080:8090 -p 40051:50051 
+$ sh docker.sh --run  --net main -p 8080:8090 -p 40051:50051 
+```
+
+#### Full node on the nile test network
+```shell
+$ sh docker.sh --run --net net
+```
+
+#### Full node on the private network
+you can also build your own private-net and will download a configuration file from the network for your private network, which will be stored in your local config/ directory.
+```shell
+$ sh docker.sh --run --net private
+```
+#### Configuration
+The script will automatically download and use the corresponding  configuration file from the github repository according to the `--net` parameter. if you don't want to update the configuration file every time you start the service, please add a startup parameter：
```

Comment:
some advice:
1. two blank spaces between `corresponding  configuration`.
2. every time when you start the service